README.md

```diff
@@ -1,121 +1,41 @@
 # dataloader-sequelize
-Batching and simplification of Sequelize with facebook/dataloader
+
+Batching, caching and simplification of Sequelize with facebook/dataloader
 
 [![Build Status](https://circleci.com/gh/mickhansen/dataloader-sequelize.svg)](https://circleci.com/gh/mickhansen/dataloader-sequelize)
 [![Coverage](https://codecov.io/gh/mickhansen/dataloader-sequelize/branch/master/graph/badge.svg)](https://codecov.io/gh/mickhansen/dataloader-sequelize)
 
-## What it does
-```js
-const User = sequelize.define('user');
-```
-
-Suppose you search for two users simultaneously (technically, within the same tick):
-
-```js
-User.findById(42);
-User.findById(5678);
-```
-
-Normally, this would result in two different queries:
-
-```sql
-SELECT ... FROM "user" WHERE "id" = 42;
-SELECT ... FROM "user" WHERE "id" = 5678;
-```
-
-However, by using `dataloader-sequelize`, the two requests will be batched into a single query:
-
-```sql
-SELECT ... FROM "user" WHERE "id" IN (42, 5678);
-```
+# How it works
 
-## Why is that a good idea?
+dataloader-sequelize is designed to provide per-request catching/batching for sequelize lookups, most likely in a graphql environment
 
-If the two `findById` calls are right next to each other, this module is semi-pointless - you
-can just change your own code into a single `findAll` call. However, if the two calls originate
-from different parts of your application, it's nice to invoke what looks like a request for a
-single row somewhere, and have it batched with other similar requests under the hood for increased
-performance.
+# API
 
-One case where this shines especially bright is in connection with graphql, specifically [graphql-sequelize](https://github.com/mickhansen/graphql-sequelize).
-Suppose we have the following schema:
+## `createContext(sequelize, object options)`
+* must not be called before all models and associations are defined
```

Comment:
At first I read this as "create context before you create associations" (the not is easy to miss)

Perhaps "Should be called after all models and associations are defined"